Makefile

```diff
@@ -117,43 +115,21 @@ release-check-blocking-prs: ## Check github for release blocking PRs
 release-bootstrap-history: ## bootstrap history for a new release
 	$(IN_VENV) python scripts/bootstrap_history.py --release $(RELEASE_CURR)
 
-npm-deps: ## Install NodeJS dependencies.
-	cd client && npm install
+node-deps: ## Install NodeJS dependencies.
+	cd client && yarn
```

Comment:
IMO it'd be really great if yarn wasn't on the path that it be installed with npm - I've been shot down about similar things before.

Reply:
I can see if we can bootstrap into installing and using a local yarn when a global one isn't available.  Not sure if that'll have problems though, I've been mainly using yarn as a complete replacement for npm, not in addition.

Reply:
My impression was this was how yarn was typically installed - though I'll definitely admit I only have this impression from a Rick and Morty meme I saw on Twitter - comparing npm's role as only useful to install yarn to that of IE - which is only useful to install Firefox. We should not let Twitter memes slow us down on this I think.

Reply:
I'm willing to try it, but it's definitely not standard.  The yarn installation docs mention it as an alternative route when necessary, but explicitly recommend against it.

Reply:
Cool beans - lets drop the subject then. Request retracted.